<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemFailed Trajectory Inference</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QTabWidget" name="visualizer">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="1">
                 <widget class="QLCDNumber" name="pharus_fps">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(164, 0, 0);</string>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::Flat</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>PHARUS FPS</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLCDNumber" name="ml_fps">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(164, 0, 0);</string>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::Flat</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>ML FPS</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Pharus Sender IP</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPlainTextEdit" name="pharus_sender_ip">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>146</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="plainText">
                 <string>127.0.0.1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Touch Designer PC IP</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPlainTextEdit" name="touch_designer_pc_ip">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>146</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="plainText">
                 <string>192.168.0.2</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>observation length</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="value">
                 <number>9</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>prediction length</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="spinBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="value">
                 <number>12</number>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>field size x, y (meter)</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="sizeConstraint">
                 <enum>QLayout::SetFixedSize</enum>
                </property>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListView" name="model_list_view">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="start_button">
              <property name="text">
               <string>start</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
